src/praisonai/praisonai/cli/main.py

```
@@ -4286,6 +4286,51 @@ def handle_direct_prompt(self, prompt):
+                should_load_rules = True
+            
+            # Check config for rules settings
+            try:
+                from praisonai.cli.configuration.loader import load_config
+                config = load_config()
+                if config.rules.auto and not getattr(self.args, 'no_rules', False):
+                    should_load_rules = True
+                    max_chars = config.rules.max_chars
+                elif getattr(self.args, 'no_rules', False):
+                    should_load_rules = False
+            except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)